core/kazoo_config/src/kapps_config_util.erl

```diff
@@ -80,3 +81,14 @@ account_schema(Config) when is_binary(Config) ->
     Name = account_schema_name(Config),
     {'ok', Schema} = kz_json_schema:load(Name),
     Schema.
+
+-spec system_config_document_schema(ne_binary()) -> kz_json:object().
+system_config_document_schema(Id) ->
+    Flat = [
```

Comment:
I think flattened JSON objects should have keys as lists of keys. I would change the raw binary keys to [<<"key">>], personally. Can you also update the `json_proplist()` type in `kazoo_json.hrl` to allow `key() | keys()`? Makes dialyzer happier when using the `scripts/dialyze-changed.bash` to dialyze.